content/applications/finance/accounting/bank/misc/interbank.rst

```diff
@@ -1,132 +1,146 @@
-couple of clicks.
-
-We will take the following example to illustrate. My company has two
-bank accounts and I want to transfer 50.000 euros from one of our bank
-accounts to the another one.
+In Odoo, internal money transfers can be made in a few clicks. All you need is at least two bank
+accounts, or two cash journals, or one bank account and one cash journal.
 
 Configuration
 =============
 
-Check your Chart of Accounts and default transfer account
----------------------------------------------------------
-
-To handle internal transfers you need a transfer account in your charts
-of account. Odoo will generate an account automatically based on the
-country of your chart of account. To configure your chart of accounts and
-check the default transfer account go into the accounting module settings,
-select :menuselection:`Configuration --> Settings`.
-
-Your chart of accounts will be pre-installed depending on the country
-specified during your registration, it cannot be changed.
-
-.. image:: interbank/interbank04.png
-   :align: center
-
-The default transfer account will automatically be generated as well
-depending on your country's legislation. If necessary it can be modified
-from the same page.
-
-.. image:: interbank/interbank05.png
-   :align: center
-
-Create a second bank account / Journal
---------------------------------------
+An internal transfer account is automatically created on your database based on your company
+localization and depending on your country’s legislation. If needed, the default :guilabel:`Internal
+transfer account` can be modified going to :menuselection:`Configuration ‣ Settings` and then under
+the :guilabel:`Default Accounts` section.
 
-Before we can register an internal transfer we need to add a new bank to
-our accounting dashboard. To do so enter the accounting module, click on
-:menuselection:`Configuration --> Bank Accounts`. Create a new bank account. You should
-fill in the **Account Number**. You can also create and edit your bank to
-specify your bank's details.
-
-.. image:: interbank/interbank03.png
-   :align: center
-
-By saving the changes you now have 2 bank accounts.
-
-.. image:: interbank/interbank06.png
-   :align: center
+.. note::
+   At least two bank accounts are needed to make internal transfers. Please refer to :ref:`this
+   documentation <interbank/import-and-reconcile>` to see how to add an additional bank account to
+   your database.
 
-Register an internal transfer from one bank to another.
-=======================================================
+Register an internal transfer from one bank to another - use case
+=================================================================
 
-We will now transfer 50.000 euros from our **Bank** to our **Bank BE57 0633
-9533 1498** account.
+Let's make an internal transfer for a company that has two bank accounts registered on its database.
+We want to make a transfer of 1.000 USD from Bank A to Bank B account.
```

Comment:
```suggestion
We want to make a transfer of 1.000 USD from Bank A to Bank B.
```
I don't think we can put the word "account " as singular, as there are two accounts (A and B). If we want to put the word "account", we should say 'from Bank A's account to Bank B's account', or 'from Bank A's account to Bank B's one', but it seems simpler just to remove the word "account".